src/lib/GOBLBuilder.svelte

```diff
@@ -1,17 +1,21 @@
 <script lang="ts">
-  import { editor, editorProblems, envelope, envelopeOrDocJSON } from "$lib/stores.js";
+  import { createEventDispatcher } from "svelte";
+  import { editor, editorProblems, envelope, envelopeAndEditorJSON } from "$lib/stores.js";
   import MenuBar from "./menubar/MenuBar.svelte";
   import Editor from "./editor/Editor.svelte";
   import { isEnvelope } from "./gobl.js";
   import { problemSeverityMap, type EditorProblem } from "./editor/EditorProblem.js";
 
+  const dispatch = createEventDispatcher();
+
   // Used for JSON Schema validation within Monaco Editor. When set, this should
   // be the JSON Schema URL of a GOBL document, e.g. an invoice. Not an envelope.
   export let jsonSchemaURL = "";
 
-  // Data is used for setting editor contents. It allows for two way component
-  // binding. E.g. when the editor contents are changed, this property is also
-  // updated.
+  // Data is used for setting editor contents. Note: there is "one way" binding;
+  // e.g. you can set data but changes are not bound to the parent. Use the
+  // `change` event, to receive changes to the editor contents and GOBL
+  // envelope.
   export let data = "";
```

Comment:
I would probably change the name now to show that it is only the initial data loaded on the editor.

Reply:
You can update the data over time though; so it's technically not only for setting initial data, but also updating it. E.g. on the demo on https://build.gobl.org the initial data is empty, and then when you pick a template we're updating the value with a JSON string.